recipes/clean-text/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "clean-text" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/clean-text-{{ version }}.tar.gz
+  sha256: 648de7c65d474c65c36ec7d1f19e815c942d67bde2db4894d3930afb75da769e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - emoji
+    - ftfy >=5.8,<6.0
```

Comment:
This failed to build in master for some reason, can you take a look: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=266046&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d

Reply:
That's weird, pipelines were all goods during PR. It's not supposed to be the same pipelines ?

Reply:
It should be identical, perhaps it's just a flaky test. I'd suggest opening a PR to see if it fails again (ping me on it once you do).

Reply:
Done https://github.com/conda-forge/ftfy-feedstock/pull/16